Solution.py

```diff
@@ -0,0 +1,43 @@
+class Solution:
+    def EvenSum (list):
+        answer = 0
+        for i in range (len(list)):
+            if list[i] % 2 == 0:
+                answer += list[i]
+        return answer
+
+    def Common (list1, list2):
```

Comment:
- This could be done with set intersection, which requires fewer lines of code. Otherwise good job.